R/makeNeighbourhoods.R

```diff
@@ -101,30 +144,37 @@ makeNeighbourhoods <- function(x, prop=0.1, k=21, refined=TRUE, seed=42, reduced
 }
 
 #' @import igraph
-.sample_vertices <- function(graph, prop, return.vertices=FALSE, seed=42){
+.sample_vertices <- function(graph, prop, seed=42){
     set.seed(seed)
     # define a set of vertices and neihbourhood centers - extract the neihbourhoods of these cells
     random.vertices <- sample(V(graph), size=floor(prop*length(V(graph))))
+    return(random.vertices)
+}
 
-    if(isTRUE(return.vertices)){
-        return(random.vertices)
-    } else{
-        message("Finding neighbours of sampled vertices")
-        vertex.list <- sapply(1:length(random.vertices), FUN=function(X) neighbors(graph, v=random.vertices[X]))
-        return(vertex.list)
+### PLotting utility function ###
+
+plotNeighborhoodSizeHist <- function(milo, bins=50){
```

Comment:
This should go into it's own separate function file with relevant documentation.

Reply:
In this case is it recommended to import using `@import ggplot2` or `@importFrom ggplot2 ggplot geom_histogram xlab theme_classic` ?

Reply:
In this case it might be okay to just import ggplot wholesale, but _I think_ the best practice is to import just the functions you explicitly need, e.g. https://github.com/LTLA/scater/blob/master/R/plotExplanatoryPCs.R